.pytool/Plugin/UncrustifyCheck/UncrustifyCheck.py

```diff
@@ -385,7 +385,7 @@ def _get_template_file_contents(self) -> None:
                 file_template_path = pathlib.Path(os.path.join(self._plugin_path, file_template_name))
                 self._file_template_contents = file_template_path.read_text()
         except KeyError:
-            logging.warn("A file header template is not specified in the config file.")
+            logging.info("A file header template is not specified in the config file.")

```

Comment:
I think we do want to get to the point we have these template files. But I agree the warning message is not necessarily helping get there.